module/data/actor/group.mjs

```diff
@@ -73,9 +77,24 @@ export default class GroupActor extends ActorDataModel.mixin(CurrencyTemplate) {
       }), { label: "DND5E.GroupMembers" }),
       attributes: new SchemaField({
         movement: new SchemaField({
-          land: new NumberField({ nullable: false, min: 0, step: 0.1, initial: 0, label: "DND5E.MovementLand" }),
-          water: new NumberField({ nullable: false, min: 0, step: 0.1, initial: 0, label: "DND5E.MovementWater" }),
-          air: new NumberField({ nullable: false, min: 0, step: 0.1, initial: 0, label: "DND5E.MovementAir" })
+          land: new NumberField({
+            nullable: false, min: 0, step: 0.1, initial: 0, speed: true, label: "DND5E.MovementLand"
+          }),
+          water: new NumberField({
+            nullable: false, min: 0, step: 0.1, initial: 0, speed: true, label: "DND5E.MovementWater"
+          }),
+          air: new NumberField({
+            nullable: false, min: 0, step: 0.1, initial: 0, speed: true, label: "DND5E.MovementAir"
+          }),
+          pace: new StringField({
+            required: true, blank: false, initial: "normal", choices: () => CONFIG.DND5E.travelPace,
+            label: "DND5E.Travel.Label"
+          }),
+          units: new StringField({
+            required: true, nullable: true, blank: false, label: "DND5E.MovementUnits", initial: () => {
+              return game.settings.get("dnd5e", "metricLengthUnits") ? "km" : "mi";
```

Comment:
We should probably add another entry to `CONFIG.DND5E.defaultUnits` for travel pace. That would allow us to use the `defaultUnits` utility method to match elsewhere.

Reply:
I suppose we could do some special casing in `defaultUnits` to use: `metric${(type === "travelPace" ? "length" : type).capitalize()}Units` for the settings key and piggyback on the normal length units. Mainly I think it would be good to have them configurable in case people want to add some special fantasy units.